composer/core/logging/logger.py

```diff
@@ -41,7 +48,7 @@ class Logger:
     :class:`~composer.core.logging.base_backend.LoggerCallback`.
```

Comment:
Please expand on this a bit more to resolve the confusing of `Logger` and `LoggerCallback`. e.g. that `Logger` is what users call from whtin their algorithms/callbacks and this routes them to a number of different loggers (e.g. file, dictionary, etc.)

Reply:
Wasn't able to get `TLogData` to resolve (see above), but i provided more explanation.